### ISSUE TITLE
Alternative text for content (images/icons) or mark as presentation

### ISSUE THREAD

**Issue by crgeary** [WDOTS/wdots.github.io#79]
A lot of the images on the site are missing `alt=""` attributes, would be nice to add them for those using screen readers, with disabled images, or any other case where images are not displayed.

If the image is used for content, like a photograph or illustration then a meaningful `alt=""` tag should be included. If for example, the image is of

If the image is used for design, or doesn't have any real meaning from a content perspective, then a `role="presentation"` attribute can be added instead of the `alt` attribute (historically an empty `alt` was used to say it was apart of design, but seeing as though we're all on modern browsers, I would say the `role` attribute is a better choice).

Here's [a guide](https://www.w3.org/WAI/tutorials/images/decorative/) for anyone wanting to take a crack at it.

If you want to checkout the site, from an accessibility perspective and to identify where these issues are, you can use the [WAVE Chrome Extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
